package.json

```diff
@@ -34,7 +34,7 @@
     "pretest": "npm run build",
     "test": "npm run test-node && npm run test-chrome",
     "test-node": "mocha --require ./test/bootstrap/index.js --reporter dot test/*.js",
-    "test-chrome": "karma start karma.conf.cjs --single-run --browsers HeadlessChrome",
+    "test-chrome": "web-test-runner",
     "test-firefox": "karma start karma.conf.cjs --browsers Firefox",
```

Comment:
should this one be updated too and remove karma entirely? 👀 maybe im just reading the pr before its ready :D

Reply:
hehe yeah the plan is the remove all of this. I just wanted to push this branch to keep it somewhere as I tend to forget random branches on my machine.